java/test/jmri/jmrix/can/cbus/node/CbusAllocateNodeNumberTest.java

```diff
@@ -104,19 +104,24 @@ public void testDisplayDialogue() {
             jtfo.setText("123");
             JLabelOperator labelOper4 = new JLabelOperator(jdo, CbusNodeConstants.getReservedModule(123));
             assertNotNull(labelOper4);
-
+            // JUnitUtil.waitFor(1000);
```

Comment:
Are these intended to be in the code? The line is added but commented out.

Reply:
Thanks for checking, these are intended to be commented out, left in for any future test debug.